docs(sql): How to change security context for SQL API (#4949)
cube-js/cube.js: docs/content/API-Reference/SQL-API.mdx

};
```

## Security Context (Row-Level Security)

> Cube.js provides security context functionality, which can be used in the

Reviewer: ```suggestion
Cube's SQL API can also use the Security Context for
```

[Dynamic Schema Creation][ref-dynamic-schemas] or [`queryRewrite`][ref-config-queryrewrite] property in your [`cube.js`
configuration file][ref-config-js].

> By default, SQL API uses current user's security context, but there is way how to change it.

Reviewer: ```suggestion
By default, the SQL API uses the current user's Security Context, but this behaviour can be modified so that certain users are allowed to switch. To do this, we must first define which user is allowed to change Security Context:
```

First, you need to define what user is allowed to change security context:

```

> CUBEJS_SQL_SUPERUSER=admin

Reviewer: We should add `CUBEJS_SQL_SUPERUSER` to https://cube.dev/docs/reference/environment-variables

Reviewer: > I didn't find a section for it. Should I create a new one?

That's a great point, could you create a new 'SQL API' section? Thank you 😀

Author: ✅

Reviewer: Amazing, thank you
